doc/manual/manual.bib

```diff
@@ -611,6 +611,16 @@ @Article{HGSLM03
   pages =	 {15--30}
 }
 
+@Article{CSG12,
+  title={A comparison of numerical surface topography calculations in geodynamic modelling: An evaluation of the ‘sticky air’method},
+  author={Crameri, F and Schmeling, H and Golabek, GJ and Duretz, T and Orendt, R and Buiter, SJH and May, DA and Kaus, BJP and Gerya, TV and Tackley, PJ},
```

Comment:
(There are a number of other inconsistent entries in manual.bib that I'm going to fix right now.)